File: all.js
const axios = require('axios');

async function connect(end_point, post) {
    try {
        const headers = {
            'User-Agent': 'Mozilla/4.0 (compatible; MSIE 5.01; Windows NT 5.0)'
        };
        const response = await axios.post(end_point, new URLSearchParams(post), { headers });
        return response.data;
    } catch (error) {
        console.error(error);
        return false;
    }
}

// Contoh menampilkan layanan
let api_url = 'https://buzzerpanel.id/api/json.php';
let post_data = {
    api_key: 'randomkey',
    secret_key: 'secret_key',
    action: 'services'
};
connect(api_url, post_data).then(api => console.log(api));

// Contoh membuat pesanan
api_url = 'api_url';
post_data = {
    api_key: 'randomkey',
    action: 'order',
    secret_key: 'secret_key',
    service: 1,
    data: 'nasa',
    quantity: 150
};
connect(api_url, post_data).then(api => console.log(api));

// Contoh mengecek status pesanan
api_url = 'api_url';
post_data = {
    api_key: 'randomkey',
    secret_key: 'secret_key',
    action: 'status',
    id: '9'
};
connect(api_url, post_data).then(api => console.log(api));

// Contoh mengecek profil akun
api_url = 'api_url';
post_data = {
    api_key: 'randomkey',
    secret_key: 'secret_key',
    action: 'profile'
};
connect(api_url, post_data).then(api => console.log(api));

File: all.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import requests
import src.config as d

def connect(end_point, post):
    headers = {'User-Agent': 'Mozilla/4.0 (compatible; MSIE 5.01; Windows NT 5.0)'}
    response = requests.post(end_point, data=post, headers=headers)
    if response.status_code == 200:
        return response.json()
    return False

def produk():
    api_url = d.api_url
    post_data = {
        'api_key': d.api_key,
        'secret_key': d.secret_key,
        'action': 'services'
    }
    api_response = connect(api_url, post_data)
    print(api_response)
        
def order():
    target = input(" [•] Masukkan alamat target : ")
    jumlah = int(input("[•] Masukkan jumlah pesanan : "))
    api_url = d.api_url
    post_data = {
        'api_key': d.api_key,
        'secret_key': d.secret_key,
        'action': 'order',
        'service': idservice,
        'data': target,
        'quantity': jumlah
    }
    send = connect(api_url, post_data)
    print(send)

def cekorder():
    api_url = d.api_url
    post_data = {
        'api_key': d.api_key,
        'secret_key': d.secret_key,
        'action': 'status',
        'id': '9'
    }
    api_response = connect(api_url, post_data)
    print(api_response)

def profile():
    api_url = d.api_url
    post_data = {
        'api_key': d.api_key,
        'secret_key': d.secret_key,
        'action': 'profile'
    }
    api_response = connect(api_url, post_data)
    print(api_response)

if __name__ == "__main__" :
    list()
